pkg/config/dynamic/middlewares.go

```diff
@@ -176,6 +176,8 @@ type Compress struct {
 	// MinResponseBodyBytes defines the minimum amount of bytes a response body must have to be compressed.
 	// Default: 1024.
 	MinResponseBodyBytes int `json:"minResponseBodyBytes,omitempty" toml:"minResponseBodyBytes,omitempty" yaml:"minResponseBodyBytes,omitempty" export:"true"`
+	// Encodings defines a list of allowed comporession algorithms
+	Encodings []string `json:"encodings,omitempty" toml:"encodings,omitempty" yaml:"encodings,omitempty" export:"true"`
```

Comment:
As you are changing the middleware configuration the Kubernetes CRDs have to be regenerated with the following command `make generate-crd`.

Reply:
Thanks, done